packages/web/src/components/inbox-unavailable-modal/InboxUnavailableModal.tsx

```diff
@@ -122,7 +129,7 @@ export const InboxUnavailableModal = ({
     <Modal
       bodyClassName={styles.modalBody}
       isOpen={isVisible}
-      onClose={onClose}
+      onClose={handleDismiss}
```

Comment:
differentiate between closing the inbox unavailable modal and "dismissing" - closing without action